pages/common/hg-log.md

```diff
@@ -0,0 +1,35 @@
+# hg log
+
+> Display the revision history of the repository.
+
+- Display the entire revision history of the repository:
+
+`hg log`
+
+- Display the revision history with an ASCII graph:
+
+`hg log --graph`
+
+- Display the revision history with names matching a specified pattern:
```

Comment:
Should we say "file names" instead of names to make it more clear ? Or do you think its good enough ?

Reply:
Sounds good on both. :+1: